lib/ruby_terraform/command_line/options.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'option'
+require_relative 'flag'
+require_relative 'boolean_option'
+
+module RubyTerraform
+  module CommandLine
+    class Options < Array
+      def initialize(option_values:, command_arguments:)
+        super()
+        @option_values = option_values
+        append_options_with_switch_overrides(command_arguments[:switch_overrides])
+        append_options(command_arguments[:standard])
+        append_boolean_options(command_arguments[:boolean])
+        append_flags(command_arguments[:flags])
```

Comment:
In the `Builder` we use the term `add` whereas here we use `append`. Does the difference mean anything in particular? Or could we use the same term in both places?

Reply:
it was simply the command being used in those methods - no other reason :)